Api/RecommendManagementInterface.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Algolia\AlgoliaSearch\Api;
+
+interface RecommendManagementInterface
```

Comment:
ManagementInterface is a standard naming convention in Magento's Service Contract design pattern. ManagementInterface is not for writing only. Any method that doesn't belong to RepositoryInterface typically goes to ManagementInterface. We can name whatever we want. I'd vote for ManagementInterface instead of TesterInterface

Reply:
You make a good point about following Magento's Service Contract patterns. `TesterInterface` was a poor example. The thing is, there are all kinds of interface naming conventions in Magento but I'm not sure this qualifies as a `ManagementInterface` since all it's really doing is reading from an API. If you think it might ever do more than that then leave it. Otherwise, something like `ProviderInterface` might be another.

Reply:
I'm a big fan of following patterns. `ManagementInterface` sounds more native to me. This is the beginning of `RecommendManagementInterface`. I envision more recommend business logics should reside in here.

Reply:
If you envision more logic here then this justifies the intent of the naming convention. [Naming things is hard!](https://martinfowler.com/bliki/TwoHardThings.html) Thanks for clarifying. 🙂